distribution/lib/Standard/Table/0.0.0-dev/src/Excel/Excel_Range.enso

```diff
@@ -162,3 +168,21 @@ type Excel_Range
 
         top_valid <| bottom_valid <|
             Excel_Range.Value (Java_Range.forRows sheet top bottom)
+
+    ## ICON data_input
+       Convert an index (1-based) to a column name
+    column_name_for_index index:Integer -> Text =
+        if index < 0 || index > excel_2007_column_limit then Error.throw (Illegal_Argument.Error "Invalid column index.") else
+            a = index % 26 + 64
+            b = if index > 26 then (index.div 26) % 26 + 64 else 0
+            c = if index > 676 then (index.div 676) % 26 + 64 else 0
+            Text.from_bytes [c, b, a]
+
+    ## ICON data_input
+       Convert a column name to an index (1-based)
+    column_index_for_name name:Text -> Integer =
+        bytes = name.bytes
+        if bytes.length < 1 || bytes.length > 3 then Error.throw (Illegal_Argument.Error "Invalid column name.") else
```

Comment:
nitpick, but can we indicate what was the invalid column name?
```suggestion
        if bytes.length < 1 || bytes.length > 3 then Error.throw (Illegal_Argument.Error "Invalid column name "+name.pretty+".") else
```

I find it much easier to debug errors where I know what was the value.